src/components/ebay-expand-button/index.js

```diff
@@ -0,0 +1,23 @@
+const assign = require('core-js-pure/features/object/assign');
+const eventUtils = require('../../common/event-utils');
+
+module.exports = require('marko-widgets').defineComponent({
+    template: require('./template.marko'),
+    getInitialState(input) {
```

Comment:
Should we add a toggle for the chevron too? IE put it here in initial state (or use input value?) and then toggle to true and change the chevron icon?
I thought that was part of the description as well for this issue?

Reply:
No, I don't believe this button should have any toggle state. Specifically, we use `aria-expanded="true"` Skin takes care of the chevron automatically.

(Note my comment in the description.)

Reply:
Also, this is prescient given how the `makeup-expander` is used, because of it's DOM manipulation outside of the components we're building. That's a huge part of keeping toggled state outside of this component. Especially when we go to replace `ebay-button` in the `ebay-menu-button` (and elsewhere).